spec/models/project_metadata_spec.rb

```diff
@@ -145,6 +145,16 @@
     end
 
     describe "#approve_project" do
+    it "Records the mediaflux id and sets the status to approved" do 
+      project_metadata = described_class.new(current_user: current_user, project:)
+      params = {mediaflux_id: 001 }
+      project_metadata.approve_project(params:) # doesn't call the doi service twice
```

Comment:
@JaymeeH what does the comment mean here?  As far as I can tell the DOI service is not called at all.
```suggestion
      project_metadata.approve_project(params:)
```

Reply:
Ah, good catch — looks like that's a comment that was left over from the create method. I can remove it